### PR TITLE
Fix Localized Strings in BPCore template

### PR DESCRIPTION
**Pull request by CasDragon**
Change LocalizedStrings in BPCore template to CopyNewest instead of Embeddded

Wolfie says Embedded breaks things, I never had issues, but Lampy did the other day, so I guess Wolfie is probably right

### CODE REVIEW

File: content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj [hunk 1]
```diff
@@ -24,7 +24,9 @@
 	</ItemGroup>
 	<ItemGroup>
 		<None Include="Info.json" CopyToOutputDirectory="PreserveNewest" />
-		<EmbeddedResource Include="LocalizedStrings.json" />
+		<None Update="LocalizedStrings.json">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</None>
```
Comment (xADDBx) [inline content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj:29]: Line above:

    <None Include="Info.json" CopyToOutputDirectory="PreserveNewest" />

Line below:

    <None Update="LocalizedStrings.json">
        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>

**Review by xADDBx — COMMENTED** [review]
Don't move the CopyToOutputDirectory into an extra line attribute; just define it inline like it's done for Info.json.
Otherwise looks good to me

File: content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj (hunk 1 above)
Comment (CasDragon) [reply content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj:29]: Blame VS for autoformatting

Comment (xADDBx) [reply content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj]: I do blame it a lot.
It tries to use 2 whitespace by default in a csproj. Imagine the horror.

**Pull request merged**